src/frontend/src/utils/pipes.ts

```diff
@@ -124,3 +124,8 @@ export const dateUndefinedPipe = (date?: Date): string => {
 export const centsToDollar = (cents: number) => {
   return (cents / 100.0).toFixed(2);
 };
+
+/** Displays a refund source as a string "Code - Name" */
+export const refundSourceToCodeName = (refundSource: ClubAccount, refundCode: number) => {
+  return `${refundCode}-${refundSource}`;
+};
```

Comment:
Everything looks good so far, you're just missing this one part of the requirements: 

<img width="728" alt="Screen Shot 2023-09-16 at 1 35 20 PM" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/60333735/3d7888fe-5b89-4bf0-ae59-6106be2dc883">

You're gonna have to do the same thing you did in ReimbursementFormView.tsx but in a different file (the one for the individual reimbursement request page). Let me know if you're confused which file this is. 

It looks like this on the frontend:
<img width="1370" alt="Screen Shot 2023-09-16 at 1 36 56 PM" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/60333735/e3218477-c261-4c3e-90ff-09bd00526ac4">